### ISSUE TITLE
Blog categories issue

### ISSUE THREAD

**Issue by cat-a-flame** [cfjedimaster/eleventy-blog-guide#5]
Hi Raymond!

Nice job there with this guide, it helped me a lot to start my blog!
Sadly I'm not an expert, and got stuck on creating categories: I wish to use both tags and categories on my posts. Every post has several tags, but only one category. I read your guide, copied the code but it just turned into chaos (rendering issues, the category page directs to 404, I can't display the category name on the postlist (home) page, etc.).
Is it possible that I can only use either tags or categories with your example? Can't have both?

Also noticed that if I have only one category, I still have to write it as an array, else the code slices the word and creates separate URL for each letter.

**Comment by cfjedimaster** [comment]
So there's a few things here, let me tackle the last one first.

So yes, you need to use an array. :) In theory it isn't a requirement - the code could see if the value is an array first, but I'm fine saying that posts in this blog must use an array, even if one item. Long story short - it would be possible to support both, but to keep things simple in this project, I'm going to require an array.

To your first question. It _is_ possible to use both tags and categories for blogs. I do that on my personal blog (raymondcamden.com). I don't see a lot of people doing that, and I wanted to keep things simple, so I went with only categories for the guide here. 

The issue with tags in Eleventy is that it uses it automatically as a way of creating collections. To avoid that, you could do what I do on my blog - I create a collection via code that is not tag driven at all. 

Long story short, what you want is definitely possible, but it's going to require big changes to the blog here.

**Comment by cfjedimaster** [comment]
I should add - if you want some help supporting tags, I'm definitely open to helping you, but obviously it would be a diviation from the blog created via the guide. :)

**Comment by cat-a-flame** [comment]
I totally understand your reasoning, it makes sense not too make it more cluttered! Separate categories are not as common than they used to, everyone is happy with using tags only.

If the repo of your personal blog is public, probably I can figure out how to achieve this. 🙏

**Comment by cfjedimaster** [comment]
It is - but it is somewhat messy (part of why I'm working on a new version) - https://github.com/cfjedimaster/raymondcamden2020/.

As this is now "off topic", I'm closing the issue, but you can email me directly and I'd be happy to help: raymondcamden@gmail.com.
